packages/libs/sdk/tsconfig.lib.json

```diff
@@ -6,15 +6,14 @@
     "target": "ES2020",
     "module": "ES2020",
     "outDir": "dist",
-    "rootDir": "src"
+    "rootDir": "."
```

Comment:
I had to modify the `rootDir` so `package.json` can be pulled in, the `includes` is set so just `src/` files are included in the build, I built the package and didn't see any extra change in size